src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__imac {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product {
+    &-name {
+      margin: 0 0 4px;
```

Comment:
If you use margin just for margin-bottom so maybe you can use smth else for set it ?) How about `margin-bottom: 4px;` - at this way it will be easiest to read you code for other devs in your future team )